core/modules/node/node.pages.inc

```diff
@@ -641,6 +640,12 @@ function node_preview_banner_form(array $form, array $form_state, Node $node, $n
   // Get the view mode to render the preview in.
   $view_mode = empty($_GET['view_mode']) ? 'full' : $_GET['view_mode'];
 
+  // We render this message here instead of in node_form_preview() in order to
+  // make it persistent across page refreshes while still in preview mode, and
+  // also while switching to different display modes.
+  backdrop_set_message(t('This is a preview, and links within the page are disabled.'), 'warning');
+  backdrop_set_message(t('Changes are stored temporarily. Click <em>Save</em> to make your changes permanent, or <em>Back to content editing</em> to make additional changes.'), 'warning');
```

Comment:
Better to phrase it: "Changes are stored temporarily. Click <em>Save</em> to make your changes permanent, or click <em>Back to content editing</em> to make additional changes." (adding second "click").

Reply:
...also good suggestion 👍, also done 😄